de.uni_heidelberg.zah.GaiaSky.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: de.uni_heidelberg.zah.GaiaSky
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk11]
+command: gaiasky
+finish-args:
+  - --socket=x11
```

Comment:
Sorry, just noticed in reviewing again but you need to add `--share=ipc` here as well to get optimal performance on X11 per https://docs.flatpak.org/en/latest/sandbox-permissions.html